### PR TITLE
fix error on single funnel step

### LINKED ISSUE #812: Single step funnel error
**Describe the bug**
Creating a funnel with a single step causes error

**To Reproduce**
1. Go to create a funnel
2. Add 1 action
3. Click save funnel

**Expected behavior**
Should not error

### PR DESCRIPTION
**Pull request by EDsCODE**
## Changes
- fixes #812 

## Checklist
- [ ] All querysets/queries filter by Team (if applicable)
- [ ] Backend tests (if applicable)

### CODE REVIEW

File: posthog/models/funnel.py [hunk 1]
```diff
@@ -112,7 +112,8 @@ def _build_query(self, query_bodies: dict):
         ON_TRUE = "ON TRUE"
         LEFT_JOIN_LATERAL = "LEFT JOIN LATERAL"
         QUERY_HEADER = "SELECT {people}, {fields} FROM "
-        LAT_JOIN_BODY = """({query}) {step} {on_true} {join}"""
+        LAT_JOIN_BODY = """({query}) {step} {on_true} {join}""" if len(query_bodies) > 1 else """({query}) {step} {on_true} """
```
Comment (fuziontech) [inline posthog/models/funnel.py:115]: good catch!

File: posthog/models/funnel.py [hunk 2]
```diff
@@ -112,7 +112,8 @@ def _build_query(self, query_bodies: dict):
...
+        print(LAT_JOIN_BODY)
```
Comment (fuziontech) [inline posthog/models/funnel.py:116]: probably want to nuke the print

**Review by fuziontech — CHANGES_REQUESTED** [review]
good catch! just remove the print

File: posthog/models/funnel.py (hunk 2 above)
Comment (EDsCODE) [reply posthog/models/funnel.py:116]: haha yeah i just realized there was a bunch of associated changes in this. gotta nuke the package lock too

**Comment by EDsCODE** [comment]
messed up refs for this closed this for #817

**Pull request closed without merging**